test/db/tools/r2pm

```diff
@@ -7,6 +7,15 @@ EOF
 EXPECT_ERR=
 RUN
 
+NAME=r2pm -H R2PM_GITDIR
+FILE=-
+CMDS=%R2PM_GITDIR=test;!r2pm -H R2PM_GITDIR
```

Comment:
r2pm should work as a cn internal command wirhout the ! Too. Can you copypste the test without the exclamation mark and check if its fine too? Thanks!

Reply:
Ok I updated the test with that.